docs/modules/panel_configuration/pages/index.adoc

```diff
@@ -19,22 +19,116 @@ This section describes the following types of panel configuration:
 [[custom-panel]]
 == Default custom panels
 
-{page-component-title} comes with several pre-configured, custom panels.
-They are described in this section.
+{page-component-title} comes with several pre-configured custom panels:
+
+* *Alarm Histogram:* A bar chart that shows the distribution of alarms by severity level.
++
+image::alarm-histogram.png["An Alarm Histogram panel displaying a horizontal bar graph. The bar graph shows the distribution of critical and major alarms.", 850]
 
-Alarm Histogram:: Bar chart that shows the distribution of alarms by severity level.
+* *Alarms Table:* A table that displays alarms.
+The query for this panel must include an `asTableSummary()` transform, otherwise the panel will not display any data.
 +
-image::alarm-histogram.png[Horizontal bar graph showing distribution of critical and major alarms, 500]
+image::alarm-panel.png["An Alarms Table panel showing two tables: one displays unacknowledged alarms, and the other displays acknowledged alarms", 850]
 
-Alarm Table:: Displays alarms in table format.
+* *Filter panel:* A panel that lets you specify data to include in other panels.
 +
-image::alarm-panel.png[Two tables: one displays unacknowledged alarms, and the other displays acknowledged alarms, 500]
+image::filter-panel.png["Blank fields in a Filter panel", 850]
 
-Filter Panel:: Panel that lets you specify data to include in other panels.
+* *Flow Histogram:* A bar chart that shows the distribution of flows data.
+The query for this panel must include an `asTableSummary()` transform, otherwise the panel will not display any data.
 +
-image::filter-panel.png[Blank fields in a filter panel, 500]
+image::flow-histogram.png["A Flow Histogram panel displaying a horizontal bar graph. The bar graph shows network flow data.", 850]
+
+For a panel to display data visualizations, you must define the following settings:
```

Comment:
This are the minimum options required to deisplay some data. Users can add any other function/transformation